clang/lib/DPCT/APINames_thrust.inc

```diff
@@ -114,12 +114,12 @@ ENTRY(thrust::operator*, thrust::operator*, true, NO_FLAG, P4, "Successful")
 ENTRY(thrust::operator+, thrust::operator+, true, NO_FLAG, P4, "Successful")
 ENTRY(thrust::operator-, thrust::operator-, true, NO_FLAG, P4, "Successful")
 ENTRY(thrust::operator/, thrust::operator/, true, NO_FLAG, P4, "Successful")
-ENTRY(thrust::operator<, thrust::operator<, false, NO_FLAG, P4, "comment")
+ENTRY(thrust::operator<, thrust::operator<, true, NO_FLAG, P4, "comment")
 ENTRY(thrust::operator<<, thrust::operator<<, false, NO_FLAG, P4, "comment")
```

Comment:
should be marked as true.
we have already supported `thrust::operator<<`

Reply:
Updated.